tests/Fixer/FunctionNotation/NullableTypeHintForDefaultNullValueFixerTest.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @requires PHP 7.1
```

Comment:
The tests should run on PHP lower than 7.1 to show that none of the test cases are changed on PHP 5.6 or 7.0, including cases that are changed on PHP 7.1+.

Reply:
@julienfalque Nullable type hints are not supported on lower php versions, a parse error is thrown https://3v4l.org/jPGvV .

Reply:
Yes, test cases that remove the `?` should not run on PHP 7.1. However test cases that add it on PHP 7.1+ should run on PHP 5.6 and 7.0 but should expect no change.

Reply:
As mentioned below, not sure how to do this one.